File: ostadyaab/SQLite.Expert.Professional.5.2.3.320.Portable/App/ProgramFiles/Scripts/Pascal/ListTables.pas
var
  Database: TSQLiteDatabase;
  Query: TSQLiteQuery;
begin
  Database := TSQLiteDatabase.Create(nil);
  try
    Database.FileName := 'C:\ProgramData\SQLite Expert\Professional 4 - 32bit\Data\dbdemos.db3';
    Database.Connected := True;
    Query := TSQLiteQuery.Create(nil);
    try
      Query.Database := Database;
      Query.SQL := 'select * from sqlite_master where type = "table"';
      Query.Open;
      while not Query.Eof do
      begin
        Writeln(Query.FieldByName('name').AsString);
        Query.Next;
      end;
      DisplayData(Query);
    finally
      Query.Free;
    end;
  finally
    Database.Free;
  end;
end.

File: ostadyaab/SQLite.Expert.Professional.5.2.3.320.Portable/App/ProgramFiles/Scripts/Lua/ListTables.lua
-- Lua sample: list all files in a database

database = sqlitedatabase:new()
database:filename("C:\\ProgramData\\SQLite Expert\\Professional 4 - 32bit\\Data\\dbdemos.db3")
database:open()
query = sqlitequery:new()
query:database(database)
query:sql("select * from sqlite_master where type = \"table\"")
query:open()
query:first()
while (query:eof() == false) do
  print(query:getfield("name"):value(), "\n")
  query:next()
end
query:free()
database:free()